### DISCUSSION TITLE
Pretraining NER and Textcat components of Healthsea

### DISCUSSION THREAD

**Discussion by shrinidhin** [explosion/spaCy#10456] [Help: Coding & Implementations] [answered]
Hello!I had a few questions about the [Healthsea](https://github.com/explosion/healthsea) project by spacy. While pre-training for initializing the tok2vec layer of NER and the clausecat component, 

1. If we are using a raw text json file for the pretraining, Is it required to use different files for both the components or would the same file work for both components?
2. Should the raw text data without annotations to be used for pre-training be different from the training and dev set?
3. I am currently building an entity+sentiment analyzer for news articles, using the logic used in healthsea.  So my raw text file will simply contain raw text of the news articles right?

Any help will be appreciated. Thank you!

**Comment by thomashacker** [comment, marked as answer]
Hello,
1. One training file should work for both components
2. There's no strict rule, I'd say that it's fine to use the raw text from both the train and dev dataset
3. Yes, a json/jsonl with raw text should work

**Reply by shrinidhin** [reply]
Thank you!
